src/components/modal/Modal.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useRef, useState } from 'react';
+import axiosBaseURL from '../../apis/axiosBaseURL';
+import InputTextArea from '../common/InputTextarea/InputTextarea';
+import ButtonBox from '../common/ButtonBox/ButtonBox';
+import messageIcon from '../../images/icons/messages.svg';
+import closeIcon from '../../images/icons/close.svg';
+import styles from './Modal.module.css';
+
+export default function Modal({ setModalOpen, id, name, image }) {
```

Comment:
```suggestion
import { useContext } from 'react';
import { SubjectDataContext } from '../../../contexts/SubjectDataContext'; //경로는 수정

export default function Modal({ setModalOpen }) {
const {id, name, imageSource} = useContext(SubjectDataContext);
```
`<Modal />` 컴포넌트의 prop으로 받아오던 id, name, image를 context로 받아와 구조 분해하여 적용해주시면 됩니다.

Reply:
성원님, 채연님이랑 해봤을 때 이 코드로도 동작이 됐었는데 왜 안되나 했더니 제가 id 값을 잘못입력해서 였습니다....ㅋㅋ
지금 코드로도 id 값 잘 불러오니 구조분해는 반영하지 않을게요!